pandas/tests/extension/test_masked.py

```diff
@@ -362,7 +362,21 @@ def check_accumulate(self, ser: pd.Series, op_name: str, skipna: bool):
         tm.assert_series_equal(result, expected)
 
     def test_loc_setitem_with_expansion_preserves_ea_index_dtype(self, data, request):
-        if data.dtype.kind == "b":
-            mark = pytest.mark.xfail(reason="GH#62344 incorrectly casts to object")
-            request.applymarker(mark)
         super().test_loc_setitem_with_expansion_preserves_ea_index_dtype(data)
+
+    @pytest.mark.parametrize(
+        "arr, values",
+        [
+            (pd.array([True, False]), [pd.NA, pd.NA]),
+            (pd.array([1, 2]), [pd.NA, pd.NA]),
+        ],
+    )
+    def test_cast_pointwise_result_all_na_respects_dtype(self, arr, values):
```

Comment:
dedent; this isn't overriding an ExtensionTest

Reply:
Thanks to you, I now understand a little more about when to use test classes. (08a5416)